### ISSUE TITLE
Missing metric name

### ISSUE THREAD

**Issue by miguel-angel-monjas** [infonova/prometheusbeat#24]
I'm using the Docker version of Prometheusbeat with ES 6.2. Prometheus is using cadvisor to gather Docker metrics. The name of the metrics does not seem to be arriving to Elasticsearch.

This is a sample as taken from Prometheus:
```
Mon, 04 Mar 2019 13:23:18 GMTP
container_spec_cpu_shares: 1024

container_label_com_docker_compose_service: prometheus,
container_label_com_docker_compose_slug: 41c1993279644d72a7854d00f3121e226fa39ed87583f23e0dc1ffae7c4237e,
container_label_com_docker_compose_oneoff: False,
container_label_maintainer: The Prometheus Authors <prometheus-developers@googlegroups.com>, 
container_label_com_docker_compose_container_number: 1,
container_label_com_docker_compose_version: 1.23.1,
container_label_com_docker_compose_config_hash: 167f95bd6d7f050d957025016fb2c77620d525554cd24ef9a3dc8c13380f35bf,
container_label_com_docker_compose_project: 01,
job: cadvisor
instance: cadvisor:8080,
id: /docker/5675a33223e73fba11cbbee3a234451814d2bced7bc3afb8da4796505d4a1c7f,
image: prom/prometheus:v2.7.1,
name: prometheus
```
However, what arrives to Elasticsearch misses the name of the metric. See the source field in the ES documents:
```
{
    "@timestamp": "2019-03-04T14:08:19.903Z",
    "value": 1024,
    "labels": {
      "id": "/docker/5675a33223e73fba11cbbee3a234451814d2bced7bc3afb8da4796505d4a1c7f",
      "image": "prom/prometheus:v2.7.1",
      "container_label_com_docker_compose_service": "prometheus",
      "container_label_com_docker_compose_slug": "41c1993279644d72a7854d00f3121e226fa39ed87583f23e0dc1ffae7c4237e",
      "instance": "cadvisor:8080",
      "container_label_com_docker_compose_oneoff": "False",
      "container_label_maintainer": "The Prometheus Authors <prometheus-developers@googlegroups.com>",
      "state": "iowaiting",
      "container_label_com_docker_compose_container_number": "1",
      "container_label_com_docker_compose_version": "1.23.1",
      "container_label_com_docker_compose_config_hash": "167f95bd6d7f050d957025016fb2c77620d525554cd24ef9a3dc8c13380f35bf",
      "container_label_com_docker_compose_project": "01",
      "job": "cadvisor"
    },
    "name": "prometheus"
}
```

`container_spec_cpu_shares` is missing. What am I doing wrong?

**Comment by miguel-angel-monjas** [comment]
Investigating a little bit more, the problem seems to lie in the fact that there is a label where the key name is `name`. Prometheusbeat takes it as the metrics name, overwritting it. Although metrics should not possibly have a label named "name", this seems to be an undesired behavior, doesn't it?

**Comment by boernd** [comment]
Yes this is not intended behavior, will try to fix it soon.

**Comment by miguel-angel-monjas** [comment]
Thanks

**Comment by boernd** [comment]
Fixed and merged into master. Available with the upcoming release (7.1.1).
